redis_cache/rediscache.py

```diff
@@ -79,29 +78,40 @@ def __init__(self,
                  password=None,
                  ssl=False,
                  client=None,
-                 namespace="SimpleCache"):
+                 namespace="SimpleCache",
+                 read_only_host=None):
 
         self.limit = limit  # No of json encoded strings to cache
         self.expire = expire  # Time to keys to expire in seconds
         self.prefix = namespace
         self.host = host
         self.port = port
         self.db = db
+        self.password = password
+        self.ssl = ssl
+        self.read_only_host = read_only_host
 
         try:
             if client:
-                self.connection = client
+                self.write_connection = client
             else:
-                self.connection = RedisConnect(host=self.host,
+                self.write_connection = RedisConnect(host=self.host,
                                                port=self.port,
                                                db=self.db,
-                                               password=password,
-                                               ssl=ssl).connect()
+                                               password=self.password,
+                                               ssl=self.ssl).connect()
+
+            self.read_connection = RedisConnect(host=self.read_only_host,
```

Comment:
please put this into the `else` clause there so if all of these variables don't exist.